### DISCUSSION TITLE
Audio and video in flashcards

### DISCUSSION THREAD

**Discussion by dustinmattison1974** [st3v3nmw/obsidian-spaced-repetition#304] [Q&A] [answered]
Is it possible to add audio or video into the fllashcards? I tried the audio using the Obsidian ![[audio1.wav]] But it doesn't work. This would be very useful because my students are learning language and I want them to also practice listening. I would rather use your plugin as opposed to going to a separate app like Anki or Mochi cards.

**Comment by st3v3nmw** [comment, marked as answer]
Audio and video support are now available in the latest release (v1.7.0).
